### ISSUE TITLE
Plugins tutorial should be improved to be more new user friendly

### ISSUE THREAD

**Issue by azeey** [gazebosim/gz-gui#643]
* **Test enviornment**
    ```text
    CPU: Ryzen 5 5600x, Ram: 32GB @ 3200 Mhz, GPU: RTX 3080
    Clean copy of Ubuntu 24.04 Humble
    ROS 2 Jazzy Jalisco full desktop
    ```
* **Summary**
     * In my humble opinion this tutorial may need to be presented in context of a more hands-on example. The current version was hard to follow, specially as a newcomer to Gazebo.
    * Add a note stating if built from source, ```GZ_GUI_PLUGIN_PATH``` can be set as follows
    ```bash
    cd ~/workspace/src/gz-gui
    export GZ_GUI_PLUGIN_PATH=${HOME}/workspace/src/gz-gui
    echo $GZ_GUI_PLUGIN_PATH
    gz gui -s ImageDisplay
    ```
    * Tutorial does not show how to set `GZ_GUI_PLUGIN_PATH`. Nor it states the user had to change direcotry into ```src/gz-gui``` directory before this command would work correctly
    

![Screenshot from 2024-09-05 20-48-49](https://github.com/user-attachments/assets/bb31b4f4-0c2e-40e1-88d9-23a30db3d866)

![Screenshot from 2024-09-05 20-49-39](https://github.com/user-attachments/assets/5aead5f2-e3b5-4a3c-9c6a-ed37b497f740)

* **Issues**
    * Users needs to change directory to ```~/workspace/src/gz-gui```. Without this step, invoking any of example commands with ```gz gui -c``` prefix throws ```XMLError``` error.
    * The tutorial page has multiple ```\subpage``` references that leads to nowhere.
    * Commands for Topic Viewer, Topic Stats, Requesters and Responders did not work.

![Screenshot from 2024-09-05 20-56-10](https://github.com/user-attachments/assets/2142b4e8-1bfb-4ed4-b6f4-7f5b95505f65)

_Originally posted by @Mechazo11 in https://github.com/gazebosim/gazebo_test_cases/issues/1323#issuecomment-2333043686_

**Comment by anshium** [comment]
Interested to take this up.

**Comment by iche033** [comment]
sure, please go ahead!

**Comment by kscottz** [comment]
@ahcorde, heads up.

**Comment by kscottz** [comment]
@anshium How are things going here? Do you need some help?

**Comment by anshium** [comment]
I am sorry, I was unwell and occupied with a few important things in the last few days. I would soon create a PR.

Apologies for the delay.
